libobs/obs-win-crash-handler.c

```diff
@@ -246,6 +246,8 @@ static inline void init_module_info(struct exception_handler_data *data)
 		data);
 }
 
+extern const char *get_win_release_id();
```

Comment:
Should this not be in a platform header?

Reply:
The function `get_win_release_id` is in `libobs/util/platform-windows.c`.  If I called it without the extern here, VS2017 gave me a warning that it was assuming this function was an extern that returned an int.  To remove the warning, I used this, which is how it is invoked when writing to the regular log as well.  See [libobs/obs-windows.c#L133](https://github.com/obsproject/obs-studio/blob/95a920d7eea8a2db117aa9ffbc062b9027f24e32/libobs/obs-windows.c#L128) and [where this usage was introduced](https://github.com/obsproject/obs-studio/commit/eadb96fbcad8a63b27da0284b4619e66202c6e8b#diff-a6289362f2f973aa4f68b59a738986cbR133) in eadb96f.

Reply:
@notr1ch replied to me out of this thread that this should be fine, so I'll resolve this conversation now so it is not left unresolved.